src/mpyl/steps/models.py

```diff
@@ -125,7 +125,7 @@ def from_configuration(run_properties: Dict, config: Dict):
             details=RunContext.from_configuration(build["run"]),
             target=Target(
                 build["parameters"].get("deploy_target", None)
-                or Target.PULL_REQUEST.value
+                or Target.PULL_REQUEST.value  # pylint: disable=E1101
```

Comment:
These warnings are actually blocking all pr's, just ran into them in my pr after rebasing. I'll add a disable I guess, but the warnings actually seem valid when I look at the code? Maybe you can explain it to me when you're back

Reply:
I fixed these in my cypress pr as well, using the `no-member` as Sam suggested. We do need to look into why these errors don't show up locally..